src/main/webapp/index.html

```diff
@@ -123,8 +123,8 @@ <h2> Create a room to vote</h2><br><br>
 
                     </table>
                 <br></br>
-                <input class="btn btn-primary" type="submit" id="btn-register" value="Click Here to Create a Room">
-                <input class="btn btn-primary" type="button" id="map-register" value="Click for a Map Room">
+                <input class="btn btn-primary" type="submit" id="btn-register" value="Click to Create a Room" hidden>
```

Comment:
Interesting approach. I did the same when getting the IP. I added hidden inputs and stuff